README.md

```diff
@@ -1 +1,176 @@
-# new-portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [hello microvers] <a name="about-project"></a>
+
+**[hello microverse]** is a portfolio project that have two parts the navgation bar and main part
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>css</summary>
+  <ul>
+    <li><a href="https://mdncss.com/">Css</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>html</summary>
+  <ul>
+    <li><a href="https://www.mdn.org/">htmlL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[use linter]**
+- **[semantic html]**
+- **[proper order]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+you don't need ant thing
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+### Install
+
+Install this project with:
+
+any installer softwere
+
+### Usage
+
+To run the project, execute the following command:
+you must clone the file first
+
+### Run tests
+
+To run tests, run the following command:
+git commands
+
+### Deployment
+
+You can deploy this project using:
+your broether
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@kirube1992](https://github.com/kirube1992)
+- Twitter: [@kirubel](https://twitter.com/kirubel08)
+- LinkedIn: [kirubel92](https://linkedin.com/in/kirubel23)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[i will add more txt]**
+- [ ] **[with css files]**
+- [ ] **[i will make it daynamic]**
+
```

Comment:
Please the `feature feature` section should be upcoming features. **Example** `Contact`, `Popup menu`, `desktop view`. What you have is okay but it shouldn't be in a sentence format